web/src/app/services/data-store/data-store.service.ts

```diff
@@ -381,15 +397,20 @@ export class DataStoreService {
 
     const predefinedLois = this.db.collection(
       `${SURVEYS_COLLECTION_NAME}/${surveyId}/lois`,
-      ref => ref.where('predefined', 'in', [true, null])
+      ref =>
+        ref.where(FieldNumbers.LocationOfInterest.source, '==', Source.IMPORTED)
     );
 
     const userLois = this.db.collection(
       `${SURVEYS_COLLECTION_NAME}/${surveyId}/lois`,
       ref =>
         ref
-          .where('predefined', '==', false)
-          .where('created.user.email', '==', userEmail)
+          .where(
+            FieldNumbers.LocationOfInterest.source,
+            '==',
+            Source.FIELD_DATA
+          )
+          .where(FieldNumbers.LocationOfInterest.owner_id, '==', userId)
```

Comment:
This and related changes will cause old submissions to stop showing once deployed. Is @lecrabe ok with us breaking existing surveys?

Reply:
We discussed. Let's merge but not deploy.